### PR TITLE
Update index.zh_CN.md

### PR DESCRIPTION
**Pull request by wupz-repo**
## Author checklist (to be completed by original Author)
- [x] Is this document a good fit for the Rocky project ?
- [x] Is this a non-English contribution? 
- [x] Title and Author MetaTags have been inserted into the document 
- [x] If applicable, steps and instructions have been tested to work on a real system
- [x] Did you perform an initial self-review to fix basic typos and grammatical correctness

## Rocky Documentation checklist  (to be completed by Rocky team) 
- [x] 1st Pass (Check that document is good fit for project and author checklist completed)
- [x] 2nd Pass (Technical Review - check for technical correctness) 
- [x] 3rd Pass (Basic Editorial Review)
- [x] 4th Pass (Detailed Editorial Review and Peer Review)
- [x] 5th Pass (Include document in TOC)
- [x] Final pass/approval (Final Review)

### CODE REVIEW

File: docs/index.zh_CN.md [hunk 1]
```diff
@@ -57,16 +61,16 @@
...
 | --- |
-| [创建完整的 LXD 服务器](guides/lxd_server.md) |
+| [Creating a full LXD Server](guides/lxd_server.md) |
 
 ## 教育/培训
 
-| 管理 | 安全 | 常规 |
-|----------------|----------|---------|
-| [系统管理](books/admin_guide/00-toc.md) | [安全实验室](labs/security/index.md) | [学习 Ansible](books/learning_ansible/index.md)
+| 管理员                                    | 安全                               | 一般                                           |
```
Comment (bobby285271) [inline docs/index.zh_CN.md:68]: Why "一般"? Is this translated by machine?

File: docs/index.zh_CN.md [hunk 2]
```diff
@@ -1,51 +1,55 @@
...
+你已经找到我们了! 欢迎来到Rocky Linux的文档中心；我们很高兴你在这里。我们有许多贡献者在添加内容，而且这些内容的缓存一直在增加。在这里你可以找到关于如何构建Rocky Linux本身的文件，以及关于对Rocky社区来说很重要的各种主题的文件。你问谁构成了这个社区？
 
-章节的顺序非常重要，但尚未最终确定，因此请随时根据需要重新排序。
+事实上，你知道的。
 
-## 操作系统安装和设置
-| Rocky 经典安装 | 在 Windows 下的 Linux 子系统上安装 Rocky |
+现在，你所看到的是我们的指南索引，其中包括设置和使用Rocky Linux的具体方法。请看右边的分类来浏览我们的内容。在上面，你可以访问一本关于安装和设置Rocky的完整书籍，以及一些高级实验室，你可以通过它来进一步了解系统管理和更多。
+
+你发现有什么遗漏吗？你是否发现了一个错误？你是否想知道如何创建一个属于你自己的文件，或者如何解决这里的问题？还记得我们说过，*你*是洛基社区吗？那么，这意味着*你*对我们很重要，我们希望你能加入我们，如果你愿意的话，并帮助使这个文档变得更好。如果你对此感兴趣，请到我们的[贡献指南](https://github.com/rocky-linux/documentation/blob/main/README.md)了解你如何能做到这一点!
```
Comment (bobby285271) [inline docs/index.zh_CN.md:9]: Maybe don't translate the word *Rocky* and leave it as is.

File: docs/index.zh_CN.md [hunk 3]
```diff
@@ -1,51 +1,55 @@
...
 ## 安全
 
 | 防火墙 | 网络安全 | 密码安全 |
 | --- | --- | --- |
-|[IPTABLES](guides/enabling_iptables_firewall.md) | [网络配置](guides/basic_network_configuration.md) | [SSH 密钥](guides/ssh_public_private_keys.md) |
-| | [SSL 密钥](guides/ssl_keys_https.md) |
-| | [生成 SSL 密钥和 Let's Encrypt](guides/generating_ssl_keys_lets_encrypt.md) |
+|[IPTABLES](guides/enabling_iptables_firewall.md) | [Networking Configuration](guides/basic_network_configuration.md) | [SSH Keys](guides/ssh_public_private_keys.md) |
+| | [SSL Keys](guides/ssl_keys_https.md) ||
+| | [Generating SSL Keys and LetsEncrypt](guides/generating_ssl_keys_lets_encrypt.md) ||
```
Comment (bobby285271) [inline docs/index.zh_CN.md:32]: Why are you converting this back to English?

**Review by bobby285271 — CHANGES_REQUESTED** [review]
This is obviously NOT an improvement of the existing one :-)

Will be great if you can withdrawn this PR and open another one when you are ready.

**Comment by justasojourner** [comment]
Hi @wupz-repo 

You may have seen that there has been discussion on the landing page for the documentation site. It was a TOC style document but has been changed to a more informational, how-to-use the site sort of document.

It has been great that you (and those working with you if any) have been maintaining the landing page in Simplified Chinese, I'm sure I speak for @sspencerwire and others in thanking you for that.

It would be great if you were able to undertake the translation of the new style home page 😄 

Also, on a technical note, I believe you are using a Windows machine, when you do a PR with the updated document `index.zh_CN.md` you *may* notice that there is now a process to convert Windows line endings [CR/LF] to [LF]. This document has information about this if you are interested to read:

https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/

Anyway you should not even notice anything as it should be automatic.

Thanks in advance for your help in translating the new home page!

**Comment by sspencerwire** [comment]
@wupz-repo Due to the new layout changes, your PR here and the work that you've done, is unfortunately no longer valid. I apologize for this. We had been working trying to figure out the best way to deal with the Table Of Contents (TOC) section of the website and finally came to the conclusion that until a better method of generating and maintaining the TOC could be found, that we were going to replace it in favor of a landing page. This was in the works when your PR was created.

We **DO** appreciate the time and effort that you've put into this and hope that you will consider doing more in the future. We are in need of your translation skills!

Thanks again for your work!!

**Pull request closed without merging**